test/release/examples/primers/errorHandling.chpl

```diff
@@ -0,0 +1,143 @@
+// Error Handling
+//
+// This primer is a short tutorial on Chapel error handling. A detailed
+// outline can be found in the :ref:`readme-errorHandling` technote.
+//
+
+/*
+  .. _primers-errorHandling-handling:
+
+  Handling Errors
+  ---------------
+
+  Standard library functions that used to halt now throw errors for
+  developers to handle as they see fit. This is especially useful for
+  recovering from errors in :mod:`IO`.
+ */
+
+use IO;
+
+const f1 = "fake.txt";
+const f2 = "still_fake.txt";
+
+var f: file;
+
+  // halts on error
+  // f = try! open(f1, iomode.r);
+
+try {
+  f = open(f1, iomode.r);
+} catch {
+  writeln("an error occurred");
+}
+
+try {
+  f = open(f1, iomode.r);
+} catch e: FileNotFoundError {
```

Comment:
Put a comment that says you can direct the catch block to only respond to errors of a certain type.